ksqldb-api-client/src/test/java/io/confluent/ksql/api/client/ClientTest.java

```diff
@@ -789,6 +791,23 @@ public void shouldHandleErrorResponseFromExecuteStatement() {
     assertThat(e.getCause().getMessage(), containsString("something bad"));
   }
 
+  @Test
+  public void shouldHandleIfNotExistsWarningResponseFromExecuteStatement() throws Exception {
```

Comment:
Can you also add a test to `ClientIntegrationTest`?

Reply:
yes